config/settings_ts/general.json

```diff
@@ -0,0 +1,32 @@
+{
+    "tooling_environment": {
+        "Name": "Tooling Account [dev]",
+        "AccountId": "405389362913",
+        "Region": "eu-central-1",
+        "EnvironmentAdminEmail": "anatolii.maslov@soname.de"
+    },
+    "monitored_environments": [
+        {
+            "Name": "Dept1 Account [dev]",
```

Comment:
I think it make sense to check naming convention for keys here. Everwhere else in this config we have this_style_names

Reply:
Don't fully get this :) What do you mean by this_style_names?

Reply:
sorry :)
I meant
"EnvironmentAdminEmail" -> "environment_admin_email"
"MetricsExtractorRoleArn" -> "metrics_extractor_role_arn"

Reply:
Got it :) will change